dask/blockwise.py

```diff
@@ -1069,26 +1063,10 @@ def optimize_blockwise(graph, keys=()):
     --------
     rewrite_blockwise
     """
-    with warnings.catch_warnings():
```

Comment:
The code that was raising this warning is no longer in `_optimize_blockwise`, so it should be safe to remove this filter